README.md

```diff
@@ -39,43 +40,48 @@ The below table lists various files used in correcting the bani portion of the d
 
 If one would like to proofread the database, they may click on the corresponding link for a PDF. The compare the proof against the source (Shabad OS Database) using the online viewer.
 
-### Bhai Nand Lal Ji
+### Sri Guru Granth Sahib Ji
 
-| Proof | Edition | Author | Publisher |
-| ----- | ------- | ------ | --------- |
-| Bhai Nand Lal Granthavali | 2009 (4th ed) | Dr. Ganda Singh | Punjabi University, Patiala |
-| Kalaam-e-Goya | 2003 (1st ed) | Pritpal Singh Bindra  | Institute of Sikh Studies, Chandigarh |
-| Rehitname | 2006 (8th ed) | Piara Singh Padam | Singh Brothers, Amritsar |
+Proof | Edition | Author | Publisher
+---|---|---|---
+[Shabadaarth (Vol. 1-4)](https://drive.google.com/open?id=1EF1MUYLPdJ_Qu2KZbp1g1lWvL1_IL2K8) | Vol. 1: October 2009<br> Vol. 2: August 2012<br> Vol. 3: July 2010<br> Vol. 4: September 2010 | - | SGPC, Sri Amritsar
```

Comment:
Split proof title into volumes with `br` tags (as done in edition column). Individually link and include which pages are referenced in each volume